src/api/v1beta1/dynakube_status.go

```diff
@@ -49,24 +59,38 @@ func GetCacheValidMessage(functionName string, lastRequestTimestamp metav1.Time,
 }
 
 type ConnectionInfoStatus struct {
-	TenantUUID  string      `json:"tenantUUID,omitempty"`
-	Endpoints   string      `json:"endpoints,omitempty"`
+	// UUID from the tenant
+	TenantUUID string `json:"tenantUUID,omitempty"`
+
+	// Available tenant endpoints
```

Comment:
```suggestion
	// Available connection endpoints
```
(this could also point to an activegate)